src/main/java/com/jfrog/ide/idea/scan/SourceCodeScannerManager.java

```diff
@@ -147,6 +188,15 @@ private void sourceCodeScanAndUpdate(ProgressIndicator indicator, Runnable check
         }
     }
 
+    private JFrogApplicationsConfig parseJFrogApplicationsConfig() throws IOException {
+        ObjectMapper mapper = createYAMLMapper();
+        File config = jfrogApplictionsConfigPath.toFile();
+        if (config.exists()) {
+            return mapper.readValue(config, JFrogApplicationsConfig.class);
+        }
+        return null;
```

Comment:
Instead of returning null, let's return a default JFrogApplicationsConfig with a single module to make sure we have 1 logic from now on